galleries/examples/specialty_plots/leftventricle_bullseye.py

```diff
@@ -104,19 +104,19 @@ def bullseye_plot(ax, data, seg_bold=None, cmap="viridis", norm=None):
 
 # Set the colormap and norm to correspond to the data for which
 # the colorbar will be used.
-cmap = mpl.cm.viridis
+cmap = mpl.colormaps["viridis"]
```

Comment:
`cmap` is now unused, and should be removed (as well as the comment about it.)

Reply:
Here we can't delete the `cmap` variable because it is used in `bullseye_plot` function and requires colormap instance, the error I am getting, 
```
 color=cmap(norm(data[start:stop])))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'str' object is not callable
```

Reply:
You are right. One could refactor this to create the colorbar within the helper function, but that's beyond the scope of this PR. Let's leave it like this for now.